packages/core/src/components/RadioButton/RadioButtonRow.tsx

```diff
@@ -46,24 +46,24 @@ const getRadioButtonAlignment = (
 };
 
 const renderLabel = (
-  value: string | React.ReactNode,
+  label: string | React.ReactNode,
   labelStyle: StyleProp<TextStyle>,
   textStyle: StyleProp<TextStyle>
 ) => {
-  if (typeof value === "string") {
-    return <Text style={[labelStyle, textStyle]}>{value}</Text>;
+  if (typeof label === "string") {
```

Comment:
are you fixing a bug here? this might be here on purpose

Reply:
@Peter only changed the name a variable for troubleshooting ... changed it back for your :)